catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -256,69 +261,72 @@ public ResourceResponse retrieveResource(URI resourceURI, Map<String, Serializab
       bytesToSkip = "0";
     }
 
-    if (resourceURI.getScheme().equals(URL_HTTP_SCHEME)
-        || resourceURI.getScheme().equals(URL_HTTPS_SCHEME)) {
-      LOGGER.debug("Resource URI is HTTP or HTTPS");
-
-      final Serializable qualifierSerializable = properties.get(ContentItem.QUALIFIER_KEYWORD);
-      if (qualifierSerializable instanceof String) {
-        final String qualifier = (String) qualifierSerializable;
-        if (StringUtils.isNotBlank(qualifier)) {
-          resourceURI =
-              UriBuilder.fromUri(resourceURI)
-                  .queryParam(ContentItem.QUALIFIER_KEYWORD, qualifier)
-                  .build();
+    switch (resourceURI.getScheme()) {
```

Comment:
This section was just updated from if/else statements to a switch statement based on Intellij recommendation.